### PR TITLE
Adjustments to the the image component #1752 

### PR DESCRIPTION
**Pull request by millianapia**

### CODE REVIEW

**Review by padms — APPROVED** [review]
LGTM

**Review by fernandolucchesi — APPROVED** [review]

**Comment by fernandolucchesi** [comment]
Hi! This has been idle for months, what is the status on this? @millianapia

File: web/pageComponents/shared/SanityImage.tsx [hunk 1]
```diff
@@ -41,14 +41,14 @@ const Image = ({
...
-      style: { ...style, objectFit: 'cover' },
+      style: { objectFit: 'cover' },
     }
   } else {
     // Layout responsive
     props = {
       width,
       height,
-      style: { ...style, display: 'flex', width: '100%', height: 'auto' },
+      style: { display: 'flex', width: '100%', height: 'auto' },
```
Comment (fernandolucchesi) [inline web/pageComponents/shared/SanityImage.tsx:51]: removing this will remove the propagation of styles everywhere on our codebase

**Review by fernandolucchesi — CHANGES_REQUESTED** [review]

File: web/pageComponents/shared/SanityImage.tsx (hunk 1 above)
Comment (millianapia) [reply web/pageComponents/shared/SanityImage.tsx:43]: there was only one place style was applied using the sanityImage, and it was redundant. Removed in the latest commit. Styling was safely removed without having any implications, this is due to the refactoring of the image component earlier last year.

**Review by fernandolucchesi — APPROVED** [review]

**Pull request closed without merging**